test/fixtures/flow/regression/issue-264/actual.js

```diff
@@ -0,0 +1,3 @@
+const map = {
+  [age <= 17] : 'Too young'
```

Comment:
Do we need the original use case? It's not important in the expected AST.

I would suggest something like that:
```js
[1 >= 0]: null
```